tutorial/creating_packages/getting_started.rst

```diff
@@ -0,0 +1,243 @@
+- It doesn't belong to the package. It only exist in the source repository, not in the package.
+
+
+Let's build the package from sources with the current default configuration, and then let the ``test_package`` folder test the package:
+
+.. code-block:: bash
+
+    $ conan create . demo/testing
+    ...
+    hello/0.1: Hello World Release!
+      hello/0.1: _M_X64 defined
+      ...
+
+If "Hello world Release!" is displayed, it worked. This is what has happened:
+
+- The *conanfile.py* together with the contents of the *src* folder have been copied (exported, in Conan terms) to the
+  local Conan cache.
+- A new build from source for the ``hello/0.1@demo/testing`` package starts, calling the ``generate()``, ``build()`` and
+  ``package()`` methods. This creates the binary package in the Conan cache.
+- Moves to the *test_package* folder and executes a ``conan install`` + ``conan build`` + ``test()`` method, to check if
+  the package was correctly created.
+
+We can now validate that the recipe and the package binary are in the cache:
+
+.. code-block:: bash
+
+    $ conan list recipes hello
+    Local Cache:
+      hello
+        hello/0.1@demo/testing#afa4685e137e7d13f2b9845987c5af77
+
+    $ conan list package-ids hello/0.1@demo/testing#afa4685e137e7d13f2b9845987c5af77
+    Local Cache:
+      hello/0.1@demo/testing#afa4685e137e7d13f2b9845987c5af77:e360b62ce00057522e221cfe56714705a46e20e2
+        settings:
+          arch=x86_64
+          build_type=Release
+          compiler=apple-clang
+          compiler.libcxx=libc++
+          compiler.version=12.0
+          os=Macos
+        options:
+          fPIC=True
+          shared=False
+
+
+The :command:`conan create` command receives the same parameters as :command:`conan install`, so
+you can pass to it the same settings and options. If we execute the following lines, we will create new package
+binaries for those configurations:
+
+.. code-block:: bash
+
+    $ conan create . demo/testing -s build_type=Debug
+    ...
+    hello/0.1: Hello World Debug!
+
+    $ conan create . demo/testing -o hello:shared=True
+    ...
+    hello/0.1: Hello World Release!
+
+
+These new package binaries will be also stored in the Conan cache, ready to be used by any project in this computer,
+we can see them with:
+
+
+.. code-block:: bash
+
+    $ conan list package-ids hello/0.1@demo/testing#afa4685e137e7d13f2b9845987c5af77
+    Local Cache:
+      hello/0.1@demo/testing#afa4685e137e7d13f2b9845987c5af77:842490321f80b0a9e1ba253d04972a72b836aa28
+        settings:
+          arch=x86_64
+          build_type=Release
+          compiler=apple-clang
+          compiler.libcxx=libc++
+          compiler.version=12.0
+          os=Macos
+        options:
+          fPIC=True
+          shared=True
+      hello/0.1@demo/testing#afa4685e137e7d13f2b9845987c5af77:a5c01fc21d2db712d56189dff69fc10f12b22375
+        settings:
+          arch=x86_64
+          build_type=Debug
+          compiler=apple-clang
```

Comment:
This was on purposely hidden in the current Getting started, to be as generic as possible, leaving only the "differences", build_type and shared

Reply:
I think it's ok to show the full output. Nothing too confusing there.